src/tools/diagnostic-messages.md

```diff
@@ -154,6 +156,40 @@ For additional details, see the
 
 [definiteAssignmentSpec](https://github.com/dart-lang/language/blob/master/resources/type-system/flow-analysis.md)
 
+### Mixin Application
```

Comment:
I'll get that updated on the analyzer side tomorrow. If you want, we can go ahead and land this in the meantime even though there will be a warning.

Reply:
Thanks! I'll hold off until it gets landed on the analyzer side.

Unfortunately GitHub actions does not seem to have a way of indicating a warning/okay failure as far as I can tell, so I am going to investigate other avenues as well. Perhaps a PR being created with the updated file, not sure yet.

Reply:
@parlough we use `experimental: true` in https://github.com/dart-lang/site-www/blob/master/.github/workflows/build.yml to make a test failure OK.

Reply:
@kwalrath I considered that but then I believe then there's no indication in the PR UI that anything failed, limiting the usefulness :(

Reply:
> To implement the issue/PR idea there would need to be a GITHUB token for an account to create it and is likely more complicated. Perhaps this check should just run when pushing to master, so it doesn't block PRs but is easy to identify? It can be a separate job so it doesn't block deploy as well. How does that sound?

I like that idea!